src/js/common/browserInfoHelper.js

```diff
@@ -59,6 +65,15 @@ export const getDeviceLocalIPAsString = () => {
   });
 };
 
+export const getDeviceLocalIPTimeStamp = () => {
```

Comment:
This function can then be removed because the timestamp would be accessed at the same time as the ip address, when calling `getDeviceLocalIPAsString` (which should probably be renamed also tbh)